automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/WorkspaceData.scala

```diff
@@ -5,4 +5,13 @@ object WorkspaceData {
   val tags = Map(
     "tag:tags" -> Seq("testing","diabetes")
   )
+
+  val tagA = Map(
+    "tag:tags" -> Seq("aTag", "cTag")
+  )
```

Comment:
maybe name this value tagsAC to represent that there are 2 tags defined?

Reply:
I can. or just remove "cTag" since test doesn't use "cTag" for now. I thought it would be good to have for future test coverage consideration.

Reply:
What do you think about moving this and `tagB` into the test itself? I'm always wary of shared test data and this is pretty simple so there's very little gain in terms of repetition. Moving it into the test insulates it from being affected by future changes and puts everything in one place so that someone reading the test doesn't have to hop around as much.